app/cms-client/src/components/Pages/Index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { loadComponent } from 'lib/Injector';
+import { Link } from 'react-router-dom';
+
+const LeftAndMain = loadComponent('LeftAndMain');
+
+const Page = () => (
+  <LeftAndMain>
+    This is the default landing page
+    <Link to="/admin/innovation/foo">Foo</Link>
```

Comment:
Ideally, the links here would know what their context is and would be relative to the root of the react admin section.